runtime/acala/src/xcm_config.rs

```diff
@@ -179,8 +181,17 @@ impl xcm_executor::Config for XcmConfig {
 		NativeTokenExistentialDeposit,
 		ExistentialDeposits,
 	>;
+	type AssetLocker = ();
+	type AssetExchanger = ();
 	type AssetClaims = PolkadotXcm;
 	type SubscriptionService = PolkadotXcm;
+	type PalletInstancesInfo = AllPalletsWithSystem;
+	type MaxAssetsIntoHolding = ConstU32<64>;
+	type FeeManager = ();
```

Comment:
I think we need to implement this to deposit the fee to treasury

Reply:
Currently Statemint/Statemine hasn't charged this fee, we will follow their implementation